File: app/src/main/java/com/sightsguru/app/data/models/Place.java
package com.sightsguru.app.data.models;

import com.google.firebase.database.IgnoreExtraProperties;
import com.google.firebase.database.PropertyName;

@IgnoreExtraProperties
public class Place {
    public String address;
    public String description;
    public double lat;
    @PropertyName("long") public double lng;
    public String title;
    @PropertyName("wiki_url") public String wikiUrl;
    @PropertyName("image_url") public String imageUrl;
    public int year;

    @Override
    public String toString() {
        return "Place{" +
                "address='" + address + '\'' +
                ", description='" + description + '\'' +
                ", lat=" + lat +
                ", lng=" + lng +
                ", title='" + title + '\'' +
                ", wikiUrl='" + wikiUrl + '\'' +
                ", imageUrl='" + imageUrl + '\'' +
                ", year=" + year +
                '}';
    }
}

File: app/src/main/java/com/sightsguru/app/data/models/Place.kt
package com.sightsguru.app.data.models

import com.google.firebase.database.IgnoreExtraProperties
import com.google.firebase.database.PropertyName
import java.io.Serializable

@IgnoreExtraProperties
data class Place(var address: String, var description: String,
                 var lat: Double, var long: Double, var title: String,
                 @PropertyName("wiki_url") var wikiUrl: String,
                 @PropertyName("image_url") var imageUrl: String,
                 var year: Int) : Serializable {
    constructor(): this("", "", 0.0, 0.0, "", "", "", 0)
}